pkg/plugin/v3/init.go

```diff
@@ -86,6 +87,8 @@ func (p *initSubcommand) BindFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&p.license, "license", "apache2",
 		"license to use to boilerplate, may be one of 'apache2', 'none'")
 	fs.StringVar(&p.owner, "owner", "", "owner to add to the copyright")
+	fs.BoolVar(&p.componentConfig, "component-config", false,
+		"create a versioned ComponentConfig file, may be 'true' or 'false'")
```

Comment:
If we will add a the new attribute that changes the scaffold then, IMO we need to persist it in the PROJECT file

Reply:
:+1: sounds good

Reply:
Done.